dali/ft/filecopy.hpp

```diff
@@ -72,6 +72,8 @@ class DALIFT_API FileFormat
     bool hasQuote() const                           { return (quote == NULL) || (*quote != '\0'); }
     bool hasQuotedTerminator() const                { return quotedTerminator; }
     const char * getFileFormatTypeString() const        { return FileFormatTypeStr[type]; }
+    bool hasJsonMarkup() const {return (rowTag.length() && *rowTag.get()=='/');} //need to add a more explicit markup indicator
+    bool hasXmlMarkup() const {return (rowTag.length() && *rowTag.get()!='/');}
 
```

Comment:
What if we use a separate flag to mark a file is JSON and doesn't change/make a magic (put an extra character when initiate the spray and remove it later, during the spray) with the rowTag? I think it will be cleaner solution. As the ECL Watch, ECL code and DFU plus also can set this flag. 
It can cause problems when the user mismatch (playing a nasty/ignorant game) the XML/JSON spray as they usually do with CSV/XML etc. files.